test/conformance/kuttl/generate/clusterpolicy/standard/data/sync/cpol-data-sync-update-trigger-no-precondition/01-assert.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kyverno.io/v2beta1
+kind: ClusterPolicy
+metadata:
+  name: cpol-data-sync-update-trigger-no-precondition
+status:
+  conditions:
+  - reason: Succeeded
+    status: "True"
+    type: Ready
```

Comment:
I copied from `cpol-data-sync-update-trigger-no-match` - because I felt like I wanted basically the same sequence of events, just with precondition instead of match.
(And I changed the policy to our use-case, because I couldn't think of a precondition to use on the ConfigMap the other test used)

I can update this for both :)

Reply:
In particular, this is still the way recommended in the scaffold: https://github.com/kyverno/kyverno/blob/main/test/conformance/kuttl/_aaa_template_resources/scaffold/policy-ready.yaml

Reply:
>Is this PR the right place to change all of those, or would it make sense as a separate cleanup piece?

Definitely not! I though there were only a few stragglers so I'll try and address this myself later.